opensrp-fp/src/main/java/org/smartregister/fp/common/util/Utils.java

```diff
@@ -1120,4 +1118,61 @@ public static void openMECWheelApp(Context context) {
         }
         context.startActivity(intent);
     }
+
+    public static LocalDate getNextContactVisitDate(JSONObject jsonObject) {
+        String methodExit = getJsonFieldValue(jsonObject, "step7", "method_exit");
```

Comment:
Please move the strings to a constants file

Reply:
Done